docs/en_US/install/3ds.md

```diff
@@ -83,19 +83,25 @@ Depending on how you installed Nimbus, launch it either through the Homebrew Lau
 
 Your selection persists between reboots.
 
-## Signing into your PNID
+## Signing in into Pretendo Network
```

Comment:
This does not flow well, and I have never seen another service phrase this as "signing in into" before? Though you could argue that `into` in the current form is also incorrect

Reply:
I'm up to whichever.

My rational with this specific change is that the term `PNID` has never been introduced in this guide and one must infer `PNID` = `Pretendo Network ID`